src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    private Random random = new Random(100);
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public String getRandomBall() {
```

Comment:
The method name `getRandomBall` suggests that it will return a `Ball` object, but it actually returns a `String`. Consider returning the actual `Ball` object and letting the caller decide how to use it.